RANes/src/fds.cpp

```diff
@@ -614,9 +614,16 @@ static int SubLoad(FCEUFILE *fp) {
 		}
 
 		FCEU_fread(diskdata[x], 1, 65500, fp);
-		memcpy(&fds_ROM[x * 65500], diskdata[x], 65500);
+
+		// Only copy the first set of subloaded data
```

Comment:
Comment could be expanded to explain why only the first set of subloaded data should be copied.

Reply:
Seems unnecessary to me, you can just scroll down a few lines to see what happens when SubLoad is called again, and this PR/commit documents it too.

Reply:
The comment reflects what the code does, not why it does it. It's almost like you did this:
```
// increment i
i++;
```
Even having read the PR, it would have been very useful for me if you had explained the changes in the code. I had to load up the project and identify what `fds_ROM` was, and find where the two calls to `SubLoad` were, before I really understood what your changes meant.

This code isn't likely to be revisited any time soon, and someone seeing it isn't going to assume it's something we added, and therefore isn't likely to look at the history to divine the intent of the functionality.

I'm just suggesting something along these lines:
```
// if firstSubLoad is true, this is the original FDS data, which is what we want to use for the RA hash. 
// if it's false, then we're looking at user save data, which we don't want to use for the RA hash.
```
I'm not saying you have to change the comment - I've already approved the review. I'm just suggesting that the comment could be expanded to better document the change without requiring the extra work of digging through the history to find the PR where the reasoning is actually documented.

Reply:
It sounds like changing the name of `fds_ROM` would be more helpful. It is based on its equivalent in `ines.cpp`.